.github/workflows/auto-image.yml

```diff
@@ -8,13 +8,17 @@ name: Create DigitalOcean Image
 jobs:
   build:
     name: Create and deploy DigitalOcean image
-    runs-on: hashicorp/packer:1.5.5
+    runs-on: ubuntu-20.04
     steps:
       - name: Build and deploy Image
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           DIGITALOCEAN_TOKEN: ${{ secrets.DIGITALOCEAN_TOKEN }}
         run: |
-          git clone https://github.com/posthog/deployment.git \
+          wget https://releases.hashicorp.com/packer/1.6.0/packer_1.6.0_linux_amd64.zip
+          && unzip packer_1.6.0_linux_amd64.zip
+          && sudo mv packer /usr/bin/
+          && sudo chmod +x /usr/bin/packer
```

Comment:
Might be wrong on this, but can't we avoid using sudo and run the packer bin in the working directory?